src/commands/rover/NicknameGroupCommand.js

```diff
@@ -24,15 +25,8 @@ class NicknameGroupCommand extends Command {
   async fn (msg, args) {
     let groupId = args.groupid
     if (groupId) {
-      if (groupId.match(/^\d+$/)) {
-        this.server.setSetting('nicknameGroup', args.groupid)
-        msg.reply(`Set nickname group to \`${args.groupid}\``)
-      } else {
-        console.log('NOPE')
-        msg.reply(`:no_entry_sign: You have attempted to bind an invalid group. Possible causes:\n\n- You forgot to put the Discord role name in quotation marks when it has spaces\n- You have attempted to bind an invalid group id. Group IDs must be a whole number or be a valid VirtualGroup name.`)
-      }
-      // this.server.setSetting('nicknameGroup', args.groupid)
-      // msg.reply(`Set nickname group to \`${args.groupid}\``)
+      this.server.setSetting('nicknameGroup', args.groupid)
```

Comment:
Variable `groupId` is set above but `args.groupid` is used in the code. Remove the `groupId` variable